mkdocs.yml

```
@@ -38,6 +38,15 @@ nav:
     - Notes: "40_development.md"
     - REST API Swagger: https://api.dandiarchive.org/swagger
     - REST API Redoc: https://api.dandiarchive.org/redoc
+    - Replicating Dandi:
+      - Getting Started: "59_getting_started_replicating_dandi.md"
+      - Initialize Vendor Accounts for DANDI: "60_initialize_vendors.md"
+      - DANDI Authentication: "61_dandi_authentication.md"
+      - DANDI CLI: "62_dandi_cli.md"
+      - DANDI Infrastructure: "63_dandi_infrastructure.md"
+      - DANDI Archive: "64_dandi_archive.md"
+      - DANDI Hub: "65_dandi_hub.md"
+      - Handling Neuroglancer in DANDI: "66_dandi_neuroglancer.md"
```

I would suggest bumping the page numbers so that they start at `60`.

```suggestion
      - Getting Started: "60_getting_started_replicating_dandi.md"
      - Initialize Vendor Accounts for DANDI: "61_initialize_vendors.md"
      - DANDI Authentication: "62_dandi_authentication.md"
      - DANDI CLI: "63_dandi_cli.md"
      - DANDI Infrastructure: "64_dandi_infrastructure.md"
      - DANDI Archive: "65_dandi_archive.md"
      - DANDI Hub: "66_dandi_hub.md"
      - Handling Neuroglancer in DANDI: "67_dandi_neuroglancer.md"
```

---

I ended up naming that file `59...` so that I didn't need to rename everything else (a long manual process, would have to re-reference the links everywhere too) -- I think this is OK for now

---

Sounds good, thanks.